src/Apps/Artist/Routes/AuctionResults/__tests__/AuctionResults.test.tsx

```diff
@@ -255,6 +259,10 @@ describe("AuctionResults", () => {
                   sort: "DATE_DESC",
                   organizations: [],
                   categories: [],
+                  createdAfterYear: "1881",
```

Comment:
The different data types for years (string vs integer) is a little challenging for my brain to parse.

Wondering if it is worth discussing modifying the response from diffusion to return a string? I think gravity uses strings to persist years as well.

Reply:
I agree that it's a bit weird. I'd much prefer to lean into having the types really match what the value is. So in this case, we know that year should always be an integer. For all the things that use year, I'd much prefer it to be a number/int/etc. 

I could probably look to consolidate this to numbers across the different usages. Let me take a peek at that.